view/theme/frio/templates/comment_item.tpl

```diff
@@ -64,3 +65,15 @@
 	<div id="comment-edit-preview-{{$id}}" class="comment-edit-preview" style="display:none;"></div>
 </div>
 
+<script>
+	$('[id=comment-fake-text-{{$id}}]').on('focus', function() {
+		dzFactory.setupDropzone('#dropzone-{{$id}}', $('#comment-edit-text-{{$id}}'), {{$max_imagesize}}); 
```

Comment:
The jQuery call is unnecessary here since it's done again in `view/js/dropzone-factory.js:12`.
```suggestion
		dzFactory.setupDropzone('#dropzone-{{$id}}', '#comment-edit-text-{{$id}}', {{$max_imagesize}}); 
```